src/components/Recipes/List.tsx

```diff
@@ -1,14 +1,27 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import Card from '../UI/Card';
 import RecipeItem from './RecipeItem';
+import { FoodprintContext } from "../../store/foodprint-context";
+
 
 const List = ({ items }: any) => {
+  const foodprintCtx = useContext(FoodprintContext);
+  const favorites = foodprintCtx.favorites.items;
+
+  const loopCheck = (item: any) => {
```

Comment:
we should move things outside a functions scope if possible, otherwise, no other file can access it (its inside a private scope of the function)